### PR TITLE
[test] Add README file for contribute CTS test case by manual

### LINKED ISSUE #1127: [doc] Add document for how to contribute manual CTS test case

### PR DESCRIPTION
**Pull request by cuiyanx**
fixed #1127

Signed-off-by: cuiyanx <yanx.cui@intel.com>

### CODE REVIEW

**Comment by cuiyanx** [comment]
@BruceDai PTAL

File: test/cts/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Contribute CTS test case
+The tests in `./test/cts/test` folder were converted from original Android CTS test, for new created CTS test file, please add them into `./test/cts/test_supplement` folder
```
Comment (BruceDai) [inline test/cts/README.md]: @cuiyanx Please move this README file into its parent  directory 'test'. And modify **./test/cts/test** to **./cts/test**, and also modify **./test/cts/test_supplement** to **./cts/test_supplement**

File: test/cts/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Steps
+Here are steps to update `./test/cts/test_supplement/cts_supplement-all.js`:
```
Comment (BruceDai) [inline test/cts/README.md]: Ditto

**Comment by BruceDai** [comment]
LGTM, thanks @cuiyanx
This README current only has CTS part, we will update it to add more details for other test cases. @ibelem  Please help merge this PR, thanks

**Pull request merged**